unpack_action.go

```diff
@@ -12,12 +13,45 @@ type UnpackAction struct {
 	File        string
 }
 
+func tarOptions(compression string) string {
+	unpackTarOpts := map[string]string{
+		"gz":    "-z",
+		"bzip2": "-j",
+		"xz":    "-J",
+	} // Trying to guess all other supported formats
+
+	return unpackTarOpts[compression]
+}
+
+func UnpackTarArchive(infile, destination, compression string, options ...string) error {
+	if err := os.MkdirAll(destination, 0755); err != nil {
+		return err
+	}
+	log.Printf("Unpacking %s\n", infile)
+
+	command := []string{"tar"}
+	command = append(command, options...)
+	command = append(command, "-x")
+	if unpackTarOpt := tarOptions(compression); len(unpackTarOpt) > 0 {
+		command = append(command, unpackTarOpt)
+	}
+	command = append(command, "-f", infile, "-C", destination)
+
+	return Command{}.Run("unpack", command...)
+}
+
+func (pf *UnpackAction) Verify(context *DebosContext) error {
+	unpackTarOpt := tarOptions(pf.Compression)
+	if len(pf.Compression) > 0 && len(unpackTarOpt) <= 0 {
```

Comment:
I find using <= 0 really weird when it's unsigned return values :)

Reply:
done. in all places with len()